### ISSUE TITLE
Question about --model in medaka pipeline

### ISSUE THREAD

**Issue by Psy-Fer** [artic-network/fieldbioinformatics#65]
Hello,

I can see in the master branch, though not in any release yet, that `--model` has been added to the medaka command.
I imagine this is from using the latest medaka versions.

In your experience is there much difference in output quality? Have you done any testing in this area?

Thanks for your time.
James

**Comment by will-rowe** [comment]
Hi @Psy-Fer - thanks for the question.

In short - we haven't tested this further than making sure the tests pass. So our test data produces the same consensus and variant set using the latest major version of medaka as it did for previous releases.

We plan on a new artic release from the master branch very soon. Once this has happened, we are going to put a bit more development into the medaka workflow and hopefully I can answer this question a bit better!

**Comment by Psy-Fer** [comment]
Awesome!

Well I just installed this repo from source, along with medaka v1.1.1  (latest) which has the new 4011 model for PromethION in it.
Conda absolutely hated me trying to install alongside my other artic installs. But I manually pushed it all through.

The medaka tests worked on the ebola set, so that seems to work well!

So now i'm running it on all of our SARS-CoV-2 samples, that I re-basecalled with the latest guppy and running the following models with `--medaka-model`

GridION/Flongle = r941_min_high_g360
PromethION = r941_prom_high_g4011

We will look at the difference to artic v1.1.1 and i think it was 3.6 or so basecalling and how model specific calling impacts SNVs and indels. Trying to answer some reviewers on our paper.
Will let you know how it goes.
